iml-gui/crate/src/lib.rs

```diff
@@ -57,6 +57,31 @@ impl Visibility {
     }
 }
 
+/// Allows for merging attributes onto an existing item
```

Comment:
Not used any more?

Reply:
Not in this PR, but I'm using it elsewhere so I figured I'd keep it here.